pantalaimon/daemon.py

```diff
@@ -850,9 +853,12 @@ def _not_json(self):
                 await self.send_ui_message(message)
 
                 try:
-                    response = await asyncio.wait_for(
-                        queue.get(), self.unverified_send_timeout
-                    )
+                    if self.send_queue:
```

Comment:
There is already a setting for this called `IgnoreVerification`. I think ignoring verification without the user configuring it explicitly isn't what we want.

Reply:
that setting doesn't do what we want: This is the unknown devices prompt we are bypassing here, which `IgnoreVerification` doesn't appear to affect. Instead, the process gets stuck.

Reply:
I lied, I failed to test this correctly - will fix.